backend/api/tests/handlers/dynamodb/get-service-clients.test.ts

```diff
@@ -14,20 +14,64 @@ describe("getServiceClientsHandler tests", () => {
 
         expect(getServiceClientsResponse).toStrictEqual({
             statusCode: 400,
-            body: JSON.stringify("No userId request parameter supplied")
+            body: "No serviceId request parameter supplied"
         });
 
         expect(getClientsSpy).not.toHaveBeenCalled();
     });
 
+    it("returns a 401 and Missing access token when no authorization header is present", async () => {
+        const testApiEvent = constructTestApiGatewayEvent({body: "", pathParameters: {serviceId: TEST_SERVICE_ID}});
+        const getServiceClientsResponse = await getServiceClientsHandler(testApiEvent);
+
+        expect(getServiceClientsResponse).toStrictEqual({
+            statusCode: 401,
+            body: "Missing access token"
+        });
+    });
+
+    it("returns a 400 when the authorization header is invalid", async () => {
+        const testApiEvent = constructTestApiGatewayEvent({
+            body: "",
+            pathParameters: {serviceId: TEST_SERVICE_ID},
+            headers: {Authorization: "Bearer invalid"}
+        });
+        const getServiceClientsResponse = await getServiceClientsHandler(testApiEvent);
+
+        expect(getServiceClientsResponse).toStrictEqual({
+            statusCode: 400,
+            body: "Invalid access token"
+        });
+    });
+
+    it("returns 403 when the user is not authorised", async () => {
+        const checkServiceUserExistsSpy = jest.spyOn(DynamoDbClient.prototype, "checkServiceUserExists").mockResolvedValue(false);
+        const testApiEvent = constructTestApiGatewayEvent({
+            body: "",
+            pathParameters: {serviceId: TEST_SERVICE_ID},
+            headers: {Authorization: "Bearer " + TEST_ACCESS_TOKEN}
```

Comment:
Looks like is a repeated generic setup. Can it be extracted out ?

Reply:
Can pull this out into a valid request constant

Reply:
Have done this now!